src/conductor/GenericDataHandler.ts

```
@@ -54,6 +72,79 @@ export class GenericDataHandler implements IDataHandler {
+        return "'llist'";
+      case DataType.PAIR:
+        return "'pair'";
+      case DataType.CLOSURE:
+        return "'function'";
+      case DataType.ANY:
+        return "'any'";
+      case DataType.OPAQUE:
+        if (value === undefined) {
+          return "'opaque'";
+        }
+        const opaque = this.opaqueMap.get(value.value as OpaqueIdentifier);
```

Biome (2.5.6)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

(lint/correctness/noSwitchDeclarations)